### PR TITLE
fix: clarify agent-run execution boundary

### LINKED ISSUE #1567: [Frontend] Describe agent-run execution boundary truthfully
## Problem

`frontend/taskdeck-web/src/views/AgentRunsView.vue` still says agent runs can be created via an automation trigger. The verified runtime creates runs only through the authenticated API path; no trigger invokes it.

## Scope

- Correct the empty-state/product copy to state the current API-created boundary.
- Keep any automation-trigger execution explicitly future-facing until a trigger call site and regression test exist.
- Add or update the focused frontend test for the rendered boundary.

## Acceptance criteria

- [ ] Agent-runs UI does not describe automation-trigger execution as shipped.
- [ ] Focused frontend test proves the truthful copy.
- [ ] Typecheck and focused test pass.

## Provenance

Non-blocking review observation on PR #1565; separate from that documentation-only diff.

## Acceptance Criteria
- [ ] Scope is confirmed against current `docs/STATUS.md`.
- [ ] Implementation includes the smallest appropriate tests or documented validation path.
- [ ] Any docs that become stale are updated in the same PR.

## Origin
Seeded as an explicit follow-up so batch execution does not silently defer discovered work.

### PR DESCRIPTION
**Pull request by Chris0Jeky**
## Summary

- Clarifies the Agent Runs empty state: current runs are created through the API, while automation-trigger execution remains future work.
- Keeps the change limited to the rendered empty-state copy and its focused view test.

## Verification

- [x] `npm test -- src/tests/views/AgentRunsView.spec.ts` — 8 passed
- [x] `npm run typecheck`
- [x] `git range-diff befaf3e7..9a5261fb b7996cf1..67356755` — patch identity preserved after base absorption
- [ ] Full frontend suite/build and Playwright were not rerun locally; hosted CI is required.

## Review follow-up

- Non-blocking P2 wording/manual observations are tracked in #1572; no fix-commit cascade was added to this two-file PR.

Closes #1567

### CODE REVIEW

File: frontend/taskdeck-web/src/views/AgentRunsView.vue [hunk 1]
```diff
@@ -112,8 +112,8 @@ function getStatusClass(status: AgentRunStatus): string {
     >
       <p class="td-agent-runs__empty-title">No runs yet</p>
       <p class="td-agent-runs__empty-body">
-        This agent has not been triggered. Runs are created when the agent is invoked
-        via the API or an automation trigger.
+        This agent has not been invoked. Runs are currently created through the API.
```
[the remark this answers is not in the document]
Comment (Chris0Jeky) [reply frontend/taskdeck-web/src/views/AgentRunsView.vue:115]: Confirmed as a non-blocking user-facing truth gap. The existing two-file #1567 slice remains bounded; #1572 now tracks the queued-record versus executed-run wording and manual alignment. Per the review budget, this does not warrant a fix-commit cascade on #1569.

File: frontend/taskdeck-web/src/views/AgentRunsView.vue [hunk 2]
```diff
@@ -112,8 +112,8 @@ function getStatusClass(status: AgentRunStatus): string {
...
+        Automation-trigger execution is planned for a future release.
```
[the remark this answers is not in the document]
Comment (Chris0Jeky) [reply frontend/taskdeck-web/src/views/AgentRunsView.vue:116]: Confirmed as a non-blocking manual/UI drift. It is tracked with the associated queued-record wording in #1572; #1569 stays limited to the already-reviewed #1567 empty-state boundary. No merge-blocking correctness, security, or data-loss path was identified.
